src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFieldInitTests.cs

```diff
@@ -689,8 +690,8 @@ public void FieldInitializerWithBadConstantValueDifferentModule()
             CreateCompilation(source2, new[] { new CSharpCompilationReference(compilation1) }, assemblyName: "2110a705-cc34-430b-9450-ca37031aa828")
                 .Emit(new System.IO.MemoryStream()).Diagnostics
                     .Verify(
-                    // error CS7038: Failed to emit module '2110a705-cc34-430b-9450-ca37031aa828'.
-                    Diagnostic(ErrorCode.ERR_ModuleEmitFailure).WithArguments("2110a705-cc34-430b-9450-ca37031aa828"));
+                    // error CS7038: Failed to emit module '2110a705-cc34-430b-9450-ca37031aa828': Unable to determine specific cause of the failure.
+                    Diagnostic(ErrorCode.ERR_ModuleEmitFailure).WithArguments("2110a705-cc34-430b-9450-ca37031aa828", CodeAnalysisResources.UnableToDetermineSpecificCauseOfFailure));
```

Comment:
>CodeAnalysisResources.UnableToDetermineSpecificCauseOfFailure) [](start = 118, length = 62)

This is likely going to cause a problem for Spanish_Unit_Tests once the strings are localized.